components/ItemGrid/LoadVideoContentTask.bs

```diff
@@ -572,7 +574,7 @@ sub addNextEpisodesToQueue(showID, additionalPartsCount as integer)
     url = Substitute("Shows/{0}/Episodes", showID)
     urlParams = { "UserId": m.global.session.user.id }
     urlParams.Append({ "StartItemId": videoID })
-    urlParams.Append({ "Limit": 50 })
+    urlParams.Append({ "Limit": queueLimit })
```

Comment:
The API's limit param includes the current episode. So if the user sets their Autoplay Episode Limit to 0, that means no next episodes will be played.

That also means if the user sets their Autoplay Episode Limit to 1, that no next episodes will be played.

That's def confusing. I work in the code and at first I was confused when testing. 😆 

I think the queueLimit should be +1 here.

The basic idea is the setting is the user telling the channel how many more episodes they want to automatically play after the one they select.

Reply:
Yeah during my testing I found the same thing and was on the fence 😛  I'll update the logic and the setting description so that's more clear

Reply:
I'll opt to keep the default setting 50 though, seems cleaner than showing 49 to the user